apps/cowswap-frontend/src/modules/permit/state/permitCacheAtom.ts

```diff
@@ -14,7 +14,7 @@ import {
  * Should never change once it has been created.
  * Used exclusively for quote requests
  */
-export const staticPermitCacheAtom = atomWithStorage<PermitCache>('staticPermitCache:v0', {})
+export const staticPermitCacheAtom = atomWithStorage<PermitCache>('staticPermitCache:v1', {})
```

Comment:
Bumping the static cache key to ignore existing permits generated with the old key.